### DISCUSSION TITLE
using opengl

### DISCUSSION THREAD

**Discussion by jbensabat** [alandefreitas/matplotplusplus#168] [Q&A]
Hello 
I have built the library with opengl
I ran the one example and it works perfectly. I would like to test some of the examples, which are dependent on gnuplot, using opengl. Is there any guiding lines on how to do that ? 
Would it be possible to provide one or two examples that depend on gnuplot adapted to opengl ?
thanks 
jac

**Comment by alandefreitas** [comment]
Hi @jbensabat ,

The OpenGL backend is experimental. It is more like a proof of concept than a complete backend. Most [`axes_object`](https://github.com/alandefreitas/matplotplusplus/tree/master/source/matplot/axes_objects)s can only generate gnuplot commands.

We are working on an alternative solution but this is going to take a while. There are a number of discussions and issues about this.
